locales/en/translation.json

```diff
@@ -1,34 +1,964 @@
+  "There are currently no transactions with Simplex": "There are currently no transactions with Simplex",
+  "Contact the Simplex support team.": "Contact the Simplex support team.",
+  "The final crypto amount you receive when the transaction is complete may differ because it is based on Wyre's exchange rate.": "The final crypto amount you receive when the transaction is complete may differ because it is based on Wyre's exchange rate.",
+  "Processing payment request": "Processing payment request",
+  "Payment method": "Payment method",
+  "Transfer ID": "Transfer ID",
+  "Blockchain Network Tx": "Blockchain Network Tx",
+  "Payment Error": "Payment Error",
+  "There was an error with the payment process through Wyre.": "There was an error with the payment process through Wyre.",
+  "Visit Help Center": "Visit Help Center",
+  "There are currently no transactions with Wyre": "There are currently no transactions with Wyre",
+  "Having problems with Wyre?": "Having problems with Wyre?",
+  "Contact the Wyre support team.": "Contact the Wyre support team.",
+  "Search Currency": "Search Currency",
+  "We couldn't find a match for ": "We couldn't find a match for ",
+  "Save Layout": "Save Layout",
+  "Home Layout": "Home Layout",
+  "Favorites": "Favorites",
+  "Light Mode": "Light Mode",
+  "System Default": "System Default",
+  "Satoshis per byte": "Satoshis per byte",
+  "within an hour": "within an hour",
+  "within hours": "within {{hours}} hours",
+  "within minutes": "within {{min}} minutes",
+  "The higher the fee, the greater the incentive a miner has to include that transaction in a block. Current fees are determined based on network load and the selected policy.": "The higher the fee, the greater the incentive a miner has to include that transaction in a block. Current fees are determined based on network load and the selected policy.",
+  "General": "General",
+  "Wallets & Keys": "Wallets & Keys",
+  "Notifications": "Notifications",
+  "Connections": "Connections",
+  "External Services": "External Services",
+  "About BitPay": "About BitPay",
+  "Log In or Sign Up": "Log In or Sign Up",
+  "Filter Gift Cards": "Filter Gift Cards",
+  "Apply Filter": "Apply Filter",
+  "Clear": "Clear",
+  "Search Gift Cards": "Search Gift Cards",
+  "Please try searching something else.": "Please try searching something else.",
+  "All Gift Cards": "All Gift Cards",
+  "My Gift Cards": "My Gift Cards",
+  "My Archived Gift Cards": "My Archived Gift Cards",
+  "See Archived": "See Archived",
+  "See Active": "See Active",
+  "You have no active gift cards.": "You have no active gift cards.",
+  "Select Country": "Select Country",
+  "Search countries": "Search countries",
+  "See all": "See all",
+  "Search for a brand": "Search for a brand",
+  "Search Results for ": "Search Results for ",
+  "Enter Email": "Enter Email",
+  "Enter Phone": "Enter Phone",
+  "Buy  Gift Card": "Buy  Gift Card",
```

Comment:
`Buy {{displayName}} Gift Cards` can you remove extra space between Buy and Gift and also we should change the key name since Buy Gift card is already available.